sound/soc/intel/boards/ehl_rt5660.c

```diff
@@ -0,0 +1,343 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ * ehl_rt5660 - ASOC Machine driver for Elkhart Lake platforms
+ * with rt5660 codec
+ */
+
+#include <linux/module.h>
+#include <linux/acpi.h>
+#include <sound/core.h>
+#include <sound/jack.h>
+#include <sound/pcm.h>
+#include <sound/pcm_params.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include <linux/list.h>
+#include <linux/device.h>
+#include <linux/gfp.h>
+#include <linux/errno.h>
+#include <linux/kernel.h>
```

Comment:
for the future: it's also recommended to sort headers alphabetically, but that's generally up to specific maintainers to reinforce. This doesn't seem to be required in `sound/` but it's still good to do. E.g. it helps avoid duplicates. Not a reason to re-submit this PR though.